README.md

```diff
@@ -46,7 +47,7 @@ Supported Docker architectures: x86_64
 
 Base OS: CentOS
 
-Latest supported MarkLogic Server version: 10.0-9.5
+Latest supported MarkLogic Server version: 11.0.0
```

Comment:
Can we just remove this line so we don't have to update this for every release? Is there value to provide this note? Users can see it in the tags, right? Perhaps we can add the "Recent Tags" card to our docker hub page to show the tags that have most recently been updated.

Reply:
Currently, there is no value to display the version in the readme.